android/src/main/kotlin/com/google/maps/flutter/navigation/GoogleMapsViewMessageHandler.kt

```diff
@@ -465,6 +473,6 @@ class GoogleMapsNavigationViewMessageHandler(
   }
 
   override fun registerOnCameraChangedListener(viewId: Long) {
-    getView(viewId.toInt()).registerOnCameraChangedListener()
+    getNavigationView(viewId.toInt()).registerOnCameraChangedListener()
```

Comment:
Should this be common for both `navigationView` and `mapView`. I think both should be able to listen camera status change events

Reply:
I thought this was part of the navigation SDK, but seems it's not. I'll move it.